### DISCUSSION TITLE
Can not exclude repo

### DISCUSSION THREAD

**Discussion by Cristishor201** [anuraghazra/github-readme-stats#1813] [Q&A] [answered]
**Describe the bug**
`exclude_repo=` doesn't works

**Expected behaviour**
It seems that at least on **fork**ed repos, like this one :D it doesn't work, on **The most used languages**.
Expected to exclude this repo, to display the other repos work.

**Additional context**
`include_forks=false` doesn't help either.

**Comment by antoninhrlt** [comment]
I've the same problem :/

**Comment by ilovelinux** [comment]
Forked repos aren´t included by default on "Most used languages" as explained in https://github.com/anuraghazra/github-readme-stats/discussions/1255

**Comment by antoninhrlt** [comment]
ok but, `&exclude_repo` does not work with my repository (that is not forked)

**Comment by rickstaa** [comment, marked as answer]
@antoninhrlt Thanks for your question. I checked your card and the following Syntax seems to work.

**Without exclusions**

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt)](https://github.com/anuraghazra/github-readme-stats)

**With exclusions**

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)

**Syntax used on your profile without exclusions**

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6)](https://github.com/anuraghazra/github-readme-stats)

**Syntax used on your profile with exclusions**

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6&hide=php,css,blade&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6&hide=php,css,blade&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)

I hope that helps.

**Comment by rickstaa** [comment]
I am closing this since I could not find any problems with the current behaviour. Please give a reproducible example below if you still think something is going wrong.

**Comment by antoninhrlt** [comment]
> @antoninhrlt Thanks for your question. I checked your card and the following Syntax seems to work.
> 
> **Without exclusions**
> 
> ```markdown
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt)](https://github.com/anuraghazra/github-readme-stats)
> ```
> 
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt)](https://github.com/anuraghazra/github-readme-stats)
> 
> **With exclusions**
> 
> ```markdown
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)
> ```
> 
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)
> 
> **Syntax used on your profile without exclusions**
> 
> ```markdown
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6)](https://github.com/anuraghazra/github-readme-stats)
> ```
> 
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6)](https://github.com/anuraghazra/github-readme-stats)
> 
> **Syntax used on your profile with exclusions**
> 
> ```markdown
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6&hide=php,css,blade&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)
> ```
> 
> [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=antoninhrlt&layout=compact&bg_color=0d1117&hide_title=true&text_color=ffffff&hide_border=true&langs_count=6&hide=php,css,blade&exclude_repo=antoninhrlt.com)](https://github.com/anuraghazra/github-readme-stats)
> 
> I hope that helps.
> 

Thank you, the problem was probably from me, maybe I've badly refreshed the page or something like that.
I'm sorry for the disturbance

**Comment by rickstaa** [comment]
@antoninhrlt No problem, happy to help!

**Comment by Cristishor201** [comment]
> Forked repos aren´t included by default on "Most used languages" as explained in #1255

Ok. But then why I can not exclude this repo ?

**Reply by rickstaa** [reply]
@Cristishor201 I'm not sure which repository you want to exclude but the following syntax should work for excluding https://github.com/Cristishor201/python-generators:

**Results without exclusion**

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Cristishor201)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Cristishor201)](https://github.com/anuraghazra/github-readme-stats)

**Results with exclusion**

```markdown
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Cristishor201&exclude_repo=python-generators)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Cristishor201&exclude_repo=python-generators)](https://github.com/anuraghazra/github-readme-stats)

Hope that helps.

**Comment by Cristishor201** [comment]
Yeah, well about that ... I guess I should apologise.

At a double check, I found that all the languages comes from my repos.
In short I seen there **Assembly**, **SWIG**, **Makefile**, **Hack**, **Shell** files, and I thought that they were come from the [github-readme-stats](https://github.com/Cristishor201/github-readme-stats) .... But it seems that I owe them in some private repos, taked from some courses, some time ago. :D

**Reply by rickstaa** [reply]
No problem happy to help.
